helmfile/values/podsecuritypolicies/common/falco.yaml.gotmpl

```diff
@@ -12,7 +12,11 @@ constraints:
         - pathPrefix: /usr
         - pathPrefix: /etc
         - pathPrefix: /dev
+        {{- if eq .Values.falco.driver.kind "ebpf" }}
+        - pathPrefix: /sys/kernel/debug
+        {{- else if eq .Values.falco.driver.kind "module" }}
         - pathPrefix: /sys/module/falco
+        {{- end }}
```

Comment:
Nothing needed here for the "modern-bpf" driver?

Reply:
No, I have tested `modern-bpf` in a cluster that supported it and using that templating seems to have worked fine. Looking in the [Falco Helm chart,](https://github.com/falcosecurity/charts/blob/master/falco/templates/pod-template.tpl#L165-L175) there doesn't seem to be any extra mounts for `modern-bpf`.